spooq/transformer/base_cleaner.py

```diff
@@ -37,8 +37,7 @@ def _add_temporary_columns(input_df, column_names, temporary_column_names):
     def _log_cleansed_values(self, input_df, column_names, temporary_column_names):
         def _only_keep_cleansed_values(col_name, temporary_col_name):
```

Comment:
```
        def _only_keep_cleansed_values(col_name, temporary_col_name):
            return (F.when(F.col(temporary_col_name) == F.col(col_name), F.lit(None))
                .otherwise(F.when(F.col(temporary_col_name).isNull() & F.col(col_name).isNull(), F.lit(None))
                .otherwise(F.when(F.col(temporary_col_name).isNull() & F.col(col_name).isNotNull(), F.lit("null"))
                .otherwise(F.col(temporary_col_name))))
            )
```

Reply:
Added the new condition.